src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -31,21 +31,45 @@ ROBOCOPY "%BUILDTOOLS_PACKAGE_DIR%\." "%TOOLRUNTIME_DIR%" /E
 set TOOLRUNTIME_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\tool-runtime\project.json
 @echo on
 call "%DOTNET_CMD%" restore "%TOOLRUNTIME_PROJECTJSON%" --source https://dotnet.myget.org/F/dotnet-core/api/v3/index.json --source https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json --source https://api.nuget.org/v3/index.json
+set RESTORE_ERROR_LEVEL=%ERRORLEVEL%
+@echo off
+if not [%RESTORE_ERROR_LEVEL%]==[0] (
+	echo ERROR: An error occured on the restore.
```

Comment:
I just want to make sure that if someone calls build.cmd and sees this error they have some idea of what they need to do to fix it.